### PR TITLE
REL-4256 Change the orientation of the slit

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2ScienceAreaGeometry.scala [hunk 1]
```diff
@@ -24,10 +24,10 @@ object Igrins2ScienceAreaGeometry extends ScienceAreaGeometry {
 
   val scienceSlitFOV: Shape =
     new Rectangle2D.Double(
-      -ScienceFovWidth.toArcsecs / 2.0,
       -ScienceFovHeight.toArcsecs / 2.0,
-      ScienceFovWidth.toArcsecs,
-      ScienceFovHeight.toArcsecs
+      -ScienceFovWidth.toArcsecs / 2.0,
+      ScienceFovHeight.toArcsecs,
+      ScienceFovWidth.toArcsecs
```
Comment (swalker2m) [inline bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2ScienceAreaGeometry.scala:30]: Just curious why not swap the values themselves?

**Review by swalker2m — COMMENTED** [review]
Looks good. :+1:

File: bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2ScienceAreaGeometry.scala (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2ScienceAreaGeometry.scala:30]: I did that first but then switcthed back, I guess height is usually larger than width for a slit?

**Review by swalker2m — APPROVED** [review]
Sorry didn't mean to make this difficult.  Looks good! :+1:

**Comment by cquiroz** [comment]
It is clearer II think this way, the slit is taller than wide at 0 position

**Pull request merged**